www/android/FileSystem.js

```diff
@@ -17,9 +17,11 @@
  * specific language governing permissions and limitations
  * under the License.
  *
-*/
+ */
 
-FILESYSTEM_PROTOCOL = 'cdvfile'; // eslint-disable-line no-undef
```

Comment:
adding a `var` keyword should have the same effect as omitting it. Using ES5 notation to retain compatibility. Same change applies to other files.

Reply:
Yes, add var.